projects/frontend/data-pipelines/gui/projects/ui/src/assets/data/appConfig.json

```diff
@@ -1,42 +1 @@
-{
```

Comment:
I find useful during troubleshooting, to log onto the container, and cat the appConfig.json. Actually used it yesterday.